manifests/plugin/memcached.pp

```diff
@@ -1,7 +1,10 @@
 # https://collectd.org/wiki/index.php/Plugin:memcached
 class collectd::plugin::memcached (
   $ensure         = 'present',
-  Hash $instances = {'default' => {'host' => '127.0.0.1', 'port' => 11211 } },
+  Hash $instances = {'default' => {
```

Comment:
Can you please enforce the structure of the hash?

Reply:
What do you mean by 'enforcing the structure' here? I'm not sure that I can follow you.